### DISCUSSION TITLE
Completing Brazilian Portguese translation

### DISCUSSION THREAD

**Discussion by EyalSeagull** [requarks/wiki#5195] [General]
Hi @NGPixel , I would like to help finishing Brazilian Portugues translation. If you can direct me with the steps I need to take, I would appreciate it. Here are some places we can complete the traslation (there are other places, too).

![image](https://user-images.githubusercontent.com/19786419/163879817-99ff146a-5b06-469f-813c-776f64d5f0f8.png)

![image](https://user-images.githubusercontent.com/19786419/163880264-c7933a8a-7689-4ab5-af33-ed4a3186e989.png)

![image](https://user-images.githubusercontent.com/19786419/163881647-4e0b83f0-e115-4162-a991-6b299727223c.png)

**Comment by NGPixel** [comment]
These strings are not localized at the moment. This will be addressed in v3.
